### PR TITLE
Change delimiter for cut command in README

### PR DESCRIPTION
**Pull request by whaley**
This should result in the asdf_update_java_home shell function in the
README to work again to aid in setting $JAVA_HOME

Addresses #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ If you want or need JAVA_HOME set you can add this to your shell initialization
 asdf_update_java_home() {
   local current
   if current=$(asdf current java); then
-    local version=$(echo $current | cut -d ' ' -f 1)
+    local version=$(echo $current | cut -d ')' -f 1)
```
Comment (rafaeltoledo) [inline README.md]: It's `(` instead, no?

Comment (whaley) [reply README.md]: Yeah, it is.  typo on my part.  I just force pushed that to be '(' instead

**Comment by brewster1134** [comment]
at least on mac, `-s` in an illegal option, but additionally, should this not be solved in the way the version is reported?  ie https://github.com/asdf-vm/asdf/blob/master/lib/commands/current.sh#L22

**Comment by rafaeltoledo** [comment]
On Mac, just use `gsed` instead of `sed` (installed using Homebrew), as the sed version embedded on MacOS is a bit old.

**Comment by brewster1134** [comment]
shouldn't this functionality be built into the asdf-java pugin, and ran whenever asdf detects a directory change and checks for .tool-versions? since this only runs on a new shell init, it has to be re-run if a different java version is required

**Pull request merged**